specification/appplatform/resource-manager/readme.ruby.md

```diff
@@ -12,6 +12,7 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
+  - tag: package-2023-12
```

Comment:
ruby lang had been deprecated, and this file can be deleted.

Reply:
I am deleting this file.